hbase-server/src/main/java/org/apache/hadoop/hbase/master/balancer/StochasticLoadBalancer.java

```diff
@@ -986,6 +990,7 @@ protected double cost() {
   static abstract class LocalityBasedCostFunction extends CostFunction {
 
     private final LocalityType type;
+    private final boolean onlyOnSsd;
```

Comment:
Should this be a favored type instead of a boolean?

Reply:
fix